DirectEmployers/MyJobs
# Allow certain CSS files to be locked to a DS-created version [PD-3576]

## What's this PR do?
Allows DS users to lock style.css and def.ui.microsite.mobile.css to versions they control.
## Screenshots (if appropriate)
It's beautiful.
![2017-08-10-150614_979x781_scrot](https://user-images.githubusercontent.com/3379265/29271040-8cb9f894-80c7-11e7-8173-59ba2ec9abc8.png)
## Where should the reviewer start?
It's a pretty simple change. Look over the template changes, make sure I didn't do anything dumb.
## How should this be manually tested?
Open up a microsite. It should look normal. Toggle "Lock CSS" in the admin. Verify that it looks as great as my screenshot. If you want to be thorough, ensure the new css links are what are defined in the ticket.
## Any background context you want to provide?
